models/topic-model.js

```diff
@@ -6,4 +6,16 @@ function fetchTopics() {
   });
 }
 
-module.exports = { fetchTopics };
+function checkTopicExists(slug) {
```

Comment:
this is fine to do (and it's well done) but if the topic doesn't exist psql would throw and error for you anyway so it's not strictly necessary, and an extra database call